### PR TITLE
[MINOR] EDA-1296 basic play method

### PR DESCRIPTION
**Pull request by srobledo-eb**
Created basic method play in Game. Added more tests related to creation of players

### CODE REVIEW

File: tests/test_game.py [hunk 1]
```diff
@@ -26,3 +28,26 @@ def test_next_turn(self, list_users, current_user, expected):
...
         self.assertEqual(next_user, expected)
+
+    @parameterized.expand([
+        [['Pedro', 'Juan'], 2],
+        [['Pedro', 'Juan', 'María', 'Ana'], 4],
+    ])
+    @patch.object(Game, 'set_first_turn')
+    @patch.object(Game, '__init__', return_value=None)
+    def test_create_players(self, list_users, len_expected, mock_constr, mock_turn):
+        with patch.object(Player, "__init__", lambda y, z: None):
```
Comment (ibrizuela-eb) [inline tests/test_game.py:39]: Change y for _self and z for name

Comment (srobledo-eb) [reply tests/test_game.py:39]: Will do in next PR

**Review by ibrizuela-eb — APPROVED** [review]
Well done Sergi!

File: tests/test_game.py [hunk 2]
```diff
@@ -12,10 +13,11 @@ def setUp(self):
         self.players_names = ['Bot1', 'Bot2']
 
     def test_create_new_game(self):
-        with patch.object(Player, "__init__", lambda y, z: None):
+        with patch.object(Player, "__init__", lambda self_arg, color_arg: None):
```
Comment (nicolasnunez-eb) [inline tests/test_game.py:16]: Why are you patching the Player init?

Comment (srobledo-eb) [reply tests/test_game.py:16]: I needed to test if the constructor of game called the constructor of Player, and I don't want the Player's constructor to execute.

I did something similar in anther test, where I didn't want a constructor to execute but needed the object of that class:

parameterized.expand([
        [['Pedro', 'Juan'], 2],
        [['Pedro', 'Juan', 'María', 'Ana'], 4],
    ])
    @patch.object(Game, 'set_first_turn')
    @patch.object(Game, '__init__', return_value=None)
    def test_create_players(self, list_users, len_expected, mock_constr, mock_turn):
        with patch.object(Player, "__init__", lambda self_arg, color_arg: None):
            game = Game()
            players = game.create_players(list_users)
            self.assertEqual(len(players), len_expected)
            mock_turn.assert_called_once_with(players)

File: tests/test_game.py [hunk 3]
```diff
@@ -26,3 +28,26 @@ def test_next_turn(self, list_users, current_user, expected):
...
+    @patch.object(Game, '__init__', return_value=None)
+    def test_create_players(self, list_users, len_expected, mock_constr, mock_turn):
+        with patch.object(Player, "__init__", lambda self_arg, color_arg: None):
+            game = Game()
+            players = game.create_players(list_users)
+            self.assertEqual(len(players), len_expected)
+            mock_turn.assert_called_once_with(players)
+
+    @patch.object(Board, 'move_pawn')
+    def test_play_calls(self, mock_place_p):
```
Comment (juanvallejo-eb) [inline tests/test_game.py:46]: maybe you could also test that the remaining moves are decreased since `play` is also doing that.

Comment (srobledo-eb) [reply tests/test_game.py:46]: That's true. I will test that in the next PR

**Pull request merged**